.github/workflows/publishRelease.yml

```
@@ -7,17 +7,17 @@ jobs:
       - name: Set up Java
-        uses: actions/setup-java@v4.7.1
+        uses: actions/setup-java@v5.0.0
         with:
           java-version: 17
           distribution: 'zulu' # openjdk
       - name: Publish package
         run: ./gradlew -Prelease publishToSonatype closeAndReleaseSonatypeStagingRepository
         env:
-           ORG_GRADLE_PROJECT_sonatypeUsername : ${{ secrets.OSSRH_USERNAME }}
-           ORG_GRADLE_PROJECT_sonatypePassword : ${{ secrets.OSSRH_PASSWORD }}
+           ORG_GRADLE_PROJECT_sonatypeUsername : ${{ secrets.ORG_GRADLE_PROJECT_sonatypeUsername }}
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)